ibm/data_source_ibm_is_operating_system_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMISOperatingSystemDataSource_basic(t *testing.T) {
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckIBMISOperatingSystemDataSourceConfig(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.ibm_is_operating_system.testacc_ds_os", "name", "red-8-amd64"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckIBMISOperatingSystemDataSourceConfig() string {
+	return fmt.Sprintf(`
+		data "ibm_is_operating_system" "testacc_ds_os" {
+			name = "red-8-amd64"
```

Comment:
can we get os name from list datasource and then pass that name here?
If `red-8-amd64` is deprecated in future, it might break this testcase